ios/RNDateTimePickerManager.m

```diff
@@ -40,4 +40,19 @@ - (UIView *)view
 RCT_REMAP_VIEW_PROPERTY(mode, datePickerMode, UIDatePickerMode)
 RCT_REMAP_VIEW_PROPERTY(timeZoneOffsetInMinutes, timeZone, NSTimeZone)
 
+RCT_CUSTOM_VIEW_PROPERTY(textColor, UIColor, RNDateTimePicker)
+{
+  UIColor* defaultColor;
+  
+  if (@available(iOS 13.0, *)) {
+      defaultColor = [UIColor labelColor];
+  } else {
+      defaultColor = [UIColor blackColor];
+  }
+    
+  UIColor* textColor = json ? [RCTConvert UIColor:json] : defaultColor;
+  [view setValue:textColor forKey:@"textColor"];
+  [view setValue:@NO forKey:@"highlightsToday"];
```

Comment:
I'm wondering - what does this do? Should we set this to `@(YES)` if the color is specified?

Reply:
> what does this do?

This is to disable the automatic highlighting for today's date 

![image](https://user-images.githubusercontent.com/4867178/76200181-24944000-622c-11ea-92e2-b6d1702233aa.png)

> Should we set this to @(YES) if the color is not specified?

Make sense! the setTextColor is now a noop when no color is specified

Reply:
sorry, I deleted this comment mistakenly, reposting

> the setTextColor is now a noop when no color is specified

this is in contrast to what was discussed before:

> this function needs to be ready to be able to cope with json representing both the case when textColor is ColorValue and when it's undefined / null in which case it should use default color.

^ this is so that when users render `DatePicker` with color=red and then re-render the component with color=undefined / null we need to change color back to default.

I didn't know what the default color was in `UIDatePicker` - it turns out it is `nil` or an actual color instance - at least on 13.3. Try running `UIColor* defaultColor = [defaultView valueForKey:@"textColor"];`. I think your previous implementation with `UIColor.labelColor` is the right way to go. For `highlightsToday` we should set the correct value too - NO when custom color is set, YES when no color is set. Meaning the implementation should look ~ like this:

```
RCT_CUSTOM_VIEW_PROPERTY(textColor, UIColor, RNDateTimePicker)
{
  UIColor* defaultColor;
  if (@available(iOS 13.0, *)) {
      defaultColor = [UIColor labelColor];
  } else {
      defaultColor = [UIColor blackColor];
  }
  UIColor* textColor = json ? [RCTConvert UIColor:json] : defaultColor;
  [view setValue:textColor forKey:@"textColor"];
  BOOL shouldHighlightToday = json;
  [view setValue:@(shouldHighlightToday) forKey:@"highlightsToday"];
}
```

does this make sense? Thanks.